ci/playbooks/config_secondary_nic_dhcp.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2017 Cable Television Laboratories, Inc. ("CableLabs")
+#                    and others.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- hosts: all
+
+  become: yes
+
+  gather_facts: no
+
+  tasks:
+    - name: Add eth1 DHCP config 50-cloud-init.yaml
+      copy:
+        dest: /etc/netplan/50-cloud-init.yaml
+        content: |2
```

Comment:
why "|2" (2 at the end)?

Reply:
It worked for indenting, before that it was breaking YAML indenting rules. You can remove if it is not necessary...